src/checkPassword.test.js

```diff
@@ -8,12 +8,37 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof checkPassword('passwordd')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('Pa$sw0rd')).toBeTruthy();
+  });
+
+  it(`should return 'false' for the valid password with 7 characters`, () => {
+    expect(checkPassword('Pa$sw0r')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the valid password with 17 characters`, () => {
+    expect(checkPassword('Pa$sw0rdPa$sw0rd1')).toBeFalsy();
+  });
 
+  it(`should return 'true' for the valid password with 16 characters`, () => {
+    expect(checkPassword('Pa$sw0rdPa$sw0rd')).toBeTruthy();
   });
 
-  // write more tests here
+  it(`should return 'false' for the password length 10 characters without 
+  special symbol`, () => {
+    expect(checkPassword('Passw0rd12')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the password length 10 characters without 
+  number`, () => {
+    expect(checkPassword('Pa$sword!!')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the password length 10 characters without 
```

Comment:
That doesn't work properly. If you will check the password 'Pa$sworд12' it will return true. Because the password contains: at least 1 digit, 1 special character, 1 uppercase letter. The non English 'д' character will be ignored.

Reply:
в оновленій версії домашки була додана валідація на кирилицю